ci/owncloud/upload_to_owncloud.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+from os import path
```

Comment:
This is unusual - normally, you do just `import os` and use `os.path.foo()` in the code. Doing it differently is going to confuse people.